src/Util.h

```diff
@@ -1191,6 +1191,26 @@ inline std::string identifier(std::string id) {
     return id;
 }
 
+/** string operation for I/O directives */
+inline std::string unescape(
+        const std::string& inputString, const std::string& needle, const std::string& replacement) {
+    std::string result = inputString;
+    size_t pos = 0;
+    while ((pos = result.find(needle, pos)) != std::string::npos) {
+        result = result.replace(pos, needle.length(), replacement);
+        pos += replacement.length();
+    }
+    return result;
+}
+
+inline std::string unescape(const std::string& inputString) {
+    std::string unescaped = unescape(inputString, "\\\"", "\"");
```

Comment:
I think these could be replaced by
```
std::string unescaped = std::regex_replace(inputString, std::regex("\\\""), "\"");
etc
```

If you do that, then you could move this whole method inside AstIO::addKVP, the only place it's used.

If you do that, then maybe this should actually be done in the parser and addKVP just adds.

Any or none of these are fine by me.